
class ExceptionInput:
    def __init__(self):
        pass
